lib/tutorials/en_US/validation.md

```diff
@@ -25,13 +25,9 @@ Validating data can be very helpful in making sure that your application is stab
 
 ## <a name="joi" /> Joi
 
-[Joi](https://github.com/hapijs/joi) is an object schema description language and validator for JavaScript objects. Joi allows you to create blueprints or schemas for JavaScript objects to ensure validation of key information. Joi can validate both input and output data. To get started with `joi`, you must first install and add it as a dependency to your project:
+[Joi](https://github.com/hapijs/joi) is an object schema description language and validator for JavaScript objects. Joi allows you to create blueprints or schemas for JavaScript objects to ensure validation of key information. Joi can validate both input and output data. Joi comes bundled with hapi, so to use it, you just have to require the module:
```

Comment:
Is that the idiomatic way to not install your own version of Joi?

Reply:
you should always install your own version.

Reply:
Yeah that's what I thought, thanks @hueniverse. Please @jarrodyellets revert it to the previous state otherwise LGTM.

Reply:
Fixing now